Topic0_3/src/ProxySet/SlowConnection.java

```diff
@@ -0,0 +1,12 @@
+package ProxySet;
+
+public class SlowConnection extends Connect{
+
+	public SlowConnection() {
+		try {
+			Thread.sleep(5000);
```

Comment:
but what makes this connection slow is the construction process not the connection itself. you need to add a sleep to connect method.